docs/changelog/4.0.rst

```diff
@@ -1,6 +1,27 @@
 Changelogs for 4.0.x
 ====================
 
+PowerDNS Authoritative Server 4.0.6
+-----------------------------------
+
+Released 6th of November 2018
+
+This release fixes PowerDNS Security Advisory
+:doc:`2018-03 <../security-advisories/powerdns-advisory-2018-03>`: Crafted zone record can cause a denial of service (CVE-2018-10851)
+
+Bug fixes
+~~~~~~~~~
+
+- `#XXXX <https://github.com/PowerDNS/pdns/pull/XXXX>`__: Crafted zone record can cause a denial of service (CVE-2018-10851)
+- `#6013 <https://github.com/PowerDNS/pdns/pull/6013>`__: Skip v6-dependent test when pdns_test_no_ipv6 is set in environment
```

Comment:
This does not need to be in here.

Reply:
Assuming this referred just to the line with #6013.

Reply:
yes indeed!